config/secrets.yml

```diff
@@ -1,8 +1,8 @@
 defaults: &defaults
-  secret_key_base:        <%= ENV['SECRET_KEY_BASE'] %>
-  omniauth_client_secret: <%= ENV['OMNIAUTH_CLIENT_SECRET'] %>
-  omniauth_client_id:     <%= ENV['OMNIAUTH_CLIENT_ID'] %>
-  devise_secret_key:      <%= ENV['DEVISE_SECRET_KEY'] %>
+  secret_key_base:        <%= Settings.secret_key_base %>
+  omniauth_client_secret: <%= Settings.omniauth_client_secret %>
+  omniauth_client_id:     <%= Settings.omniauth_client_id %>
+  devise_secret_key:      <%= Settings.devise_secret_key %>
```

Comment:
what what? I thought that Settings got its values from yml files, not the other way around